docbook/gnc-locale-de.dtd

```diff
@@ -27,46 +27,46 @@
 <!ENTITY % gm.view "<guimenu><accel>A</accel>nsicht</guimenu>">
 
 
-<!-- Transactions-Menu -->
-<!ENTITY % gm.trans "<guimenu><accel>B</accel>uchungen</guimenu>">
-
-
 <!-- Actions-Menu -->
 <!ENTITY % gm.act "<guimenu>A<accel>k</accel>tion</guimenu>">
 
 
+<!-- Transactions-Menu -->
+<!ENTITY % gm.trans "<guimenu><accel>B</accel>uchungen</guimenu>">
+
+
 <!-- Business-Menu -->
-<!ENTITY % gm.busi "<guimenu><accel>G</accel>eschäft</guimenu>">
+<!ENTITY % gm.bus "<guimenu><accel>G</accel>eschäft</guimenu>">
 
 
 <!-- Reports-Menu -->
-<!ENTITY % gm.report "<guimenu>Be<accel>r</accel>ichte</guimenu>">
-<!ENTITY % gsm.budget "<guisubmenu>B<accel>u</accel>dget</guisubmenu>">
-<!ENTITY % gmi.budget-rep "<guimenuitem>Budget-Bericht</guimenuitem>">
+<!ENTITY % gm.rep "<guimenu>Be<accel>r</accel>ichte</guimenu>">
+<!ENTITY % gsm.rep.bgt "<guisubmenu>B<accel>u</accel>dget</guisubmenu>">
+<!ENTITY % gmi.rep.bgt.bgt-rep "<guimenuitem>Budget-Bericht</guimenuitem>">
 
 
 <!-- Tools-Menu -->
 <!ENTITY % gm.tool "<guimenu><accel>W</accel>erkzeuge</guimenu>">
-<!ENTITY % gmi.aqb "<guimenuitem><accel>O</accel>nlinebanking Einrichtung…</guimenuitem>">
-<!ENTITY % gmi.price "<guimenuitem><accel>K</accel>urs Datenbank</guimenuitem>">
-<!ENTITY % gmi.sec "<guimenuitem><accel>W</accel>ertpapier Editor</guimenuitem>">
-<!ENTITY % gmi.loan "<guimenuitem><accel>D</accel>arlehensrechner</guimenuitem>">
-<!ENTITY % gmi.clbok "<guimenuitem>Buch<accel>a</accel>bschluss</guimenuitem>">
-<!ENTITY % gmi.impmap "<guimenuitem><accel>I</accel>mport-Zuordnungen Editor</guimenuitem>">
-<!ENTITY % gmi.doclink "<guimenuitem>Buchungs<accel>v</accel>erknüpfungen</guimenuitem>">
-<!ENTITY % gmi.genjour "<guimenuitem>Hauptbuch <accel>J</accel>ournal</guimenuitem>">
+<!ENTITY % gmi.to.aqb "<guimenuitem><accel>O</accel>nlinebanking Einrichtung…</guimenuitem>">
+<!ENTITY % gmi.to.price "<guimenuitem><accel>K</accel>urs Datenbank</guimenuitem>">
+<!ENTITY % gmi.to.sec "<guimenuitem><accel>W</accel>ertpapier Editor</guimenuitem>">
+<!ENTITY % gmi.to.loan "<guimenuitem><accel>D</accel>arlehensrechner</guimenuitem>">
+<!ENTITY % gmi.to.clbok "<guimenuitem>Buch<accel>a</accel>bschluss</guimenuitem>">
+<!ENTITY % gmi.to.imp-map "<guimenuitem><accel>I</accel>mport-Zuordnungen Editor</guimenuitem>">
+<!ENTITY % gmi.to.link-doc "<guimenuitem>Buchungs<accel>v</accel>erknüpfungen</guimenuitem>">
+<!ENTITY % gmi.to.gen-jour "<guimenuitem>Hauptbuch <accel>J</accel>ournal</guimenuitem>">
 
 <!-- Window-Menu -->
 <!ENTITY % gm.win "<guimenu><accel>F</accel>enster</guimenu>">
-<!ENTITY % gmi.nwin "<guimenuitem><accel>N</accel>eues Fenster</guimenuitem>">
-<!ENTITY % gmi.npage "<guimenuitem>Neues Fenster mit <accel>S</accel>eite</guimenuitem>">
+<!ENTITY % gmi.win.win "<guimenuitem><accel>N</accel>eues Fenster</guimenuitem>">
+<!ENTITY % gmi.win.page "<guimenuitem>Neues Fenster mit <accel>S</accel>eite</guimenuitem>">
 
 <!-- Help-Menu -->
 <!ENTITY % gm.help "<guimenu><accel>H</accel>ilfe</guimenu>">
```

Comment:
How about `gm.hlp`?
`grep -inr 'gm.help' .` returns only the dtds. For the entries below
`grep -inrF '.help.' .` returns also /manual/de/Help_ch_GUIMenus.xml

Reply:
Correct, this was only meant to be a test to see if it would work. The actual translation and further work on the individual chapters will come with a later commit.